plans/tier1.fmf

```diff
@@ -1,330 +0,0 @@
-/activation_key:
```

Comment:
I expect this will be reverted in final version

Reply:
All the tier1 tests are back.